styleguide/components/41-organisms/header/README.md

```diff
@@ -1,19 +1,100 @@
 # Header
 
-## General
+## When to use this component
+
+All websites and all pages must include the header.
+
+## How it works
+
+The header is the first visible component on every page. The position of the header is fixed on the page.
+
+Depending on the device's screen resolution, the header switches to one of the two following modes:
+
+* **Desktop header**: Desktop resolutions and tablet in landscape mode
+* **Mobile header**: Tablet in portrait mode and mobile resolutions
+
+### Desktop header
+
+The desktop header is used on desktop resolutions and tablets in landscape mode.
+
+The desktop header can consist of four parts:
+
+1. Top header with Ghent logo (always required)
+2. Menu with main navigation (optional)
+3. Breadcrumbs (required for most websites)
+4. Header image (optional)
+
+The top header with Ghent logo has the following components:
+
+* Ghent logo (always required)
+* Site title (optional, only for separate websites)
+* Search (optional, only if applicable)
+* Mijn Gent (optional, only if applicable)
+* Language switch (optional, only for multilingual websites with synchronous translations)
+
+When there is no header image, the light blue Ghent-style seperator is shown. When there is a header image, the light blue Ghent-style separator is not shown.
+
+**Further specific requirements for the City of Ghent website and for separate websites are listed below.**
+
+### Mobile header
+
+The mobile header is used on tablet in portrait mode and below (mobile resolutions).
+
+The mobile header can consist of three parts:
+
+1. Top header with Ghent logo (always required)
+2. Site title (optional, only for separate websites)
+3. Breadcrumbs (required for most websites)
+
+The top header with Ghent logo has the following components:
+
+* Ghent logo (always required)
+* Search button with link (optional, only if applicable)
+* Mijn Gent (optional, only if applicable)
+* Menu button (optional, only if applicable)
+* Language switch (optional, only if applicable and for multilingual websites with synchronous translations)
+
+For the top part of the mobile header, the following conditions apply:
+
+1. Where there is a header image, the header image in the mobile header is hidden. The light blue Ghent-style separator is always shown.
+
+2. When the website has a search function, the search function is always shown in the header.
+
+3. When the website has main navigation, the menu button is always shown in the header. The menu button opens the mobile menu with the main navigation.
+
+4. When the website has main navigation and also Mijn Gent and/or a language switch, Mijn Gent and/or the language switch are intergrated in the menu. The menu button is always shown in the header. The menu button opens the mobile menu with the main navigation and Mijn Gent and/or the language switch.
```

Comment:
Best add a todo and Jira ticket.
As this is not yet reality.

Reply:
@delrueba The documentation already describes the wanted behavior. Could you make a to do / JIRA ticket for this?

Reply:
@mattiasberlamont I think it's best you do it, as it's not the developers who are requesting this change.
I see designs for LEZ with login and menu in the mobile header.
I see designs for stad.gent with username in the mobile header and I see designs with user img in the mobile header.
I haven't found one with 'aanmelden' in the mobile header.

As you say above 
> Documentation is something that is constantly improving and will "never be finished" 

So I suggest we keep the documentation in line with what is actually developed and validated.

...

Reply:
@delrueba I will check and updated the designs for the header where necessary. I will create a JIRA ticket. I agree that documentation should follow development once a style guide is alive and public. For the header, I will keep it this way though, it's not efficient to adapt the documentation at this point back to describe the old behavior and then back to the new behavior when it is implemented like that.